hub/integrations/pytorch/dataset.py

```diff
@@ -244,7 +245,7 @@ def __iter__(self):
 
         while any(self.active_workers):
             try:
-                wid, data = self.data_queue.get(timeout=5)
+                wid, data = self.data_queue.get(timeout=PYTORCH_DATALOADER_TIMEOUT)
```

Comment:
can we make this an argument that the user passes and default to this value?

Reply:
I think ideally there should be no timeout and the workers should be able to report exceptions to the main thread.. so better not change API right now.